docs/configuration.md

```diff
@@ -0,0 +1,103 @@
+# Configuration
+
+
+* [UCM Configuration](#ucm-configuration)
+    * [`UNISON_DEBUG`](#unison_debug)
+    * [`UNISON_PAGER`](#unison_pager)
+    * [`UNISON_LSP_PORT`](#unison_lsp_port)
+    * [`UNISON_SHARE_HOST`](#unison_share_host)
+    * [`UNISON_SHARE_ACCESS_TOKEN`](#unison_share_access_token)
+    * [Local Codebase Server](#local-codebase-server)
+* [Codebase Configuration](#codebase-configuration)
+
+## UCM Configuration
+
+### `UNISON_DEBUG`
+
+Enable debugging output for various portions of the application. 
+See `lib/unison-prelude/src/Unison/Debug.hs` for the full list of supported flags.
+
+E.g.
+
+```sh
+# Enable ALL debugging flags (likely quite noisy)
+$ UNISON_DEBUG= ucm
```

Comment:
Side note, this probably should disable all debugging, not enable all. I'd suggest `UNISON_DEBUG=all` for all.

Reply:
I don't really care either way, @mitchellwrosen requested this behaviour so maybe talk it over between the two of you.